### PR TITLE
SAMZA-2679: Add missing Override annotations

### PR DESCRIPTION
**Pull request by bringhurst**

### CODE REVIEW

File: samza-api/src/main/java/org/apache/samza/checkpoint/CheckpointV1.java [hunk 1]
```diff
@@ -44,7 +44,7 @@ public CheckpointV1(Map<SystemStreamPartition, String> offsets) {
     this.offsets = offsets;
   }
 
-  public short getVersion() {
+  @Override public short getVersion() {
```
Comment (cameronlee314) [inline samza-api/src/main/java/org/apache/samza/checkpoint/CheckpointV1.java:47]: Looks like the style in the other parts of code are to have `@Override` on a new line above the method. i.e.:
```
@Override
public short getVersion() {
```
Sorry to request this, but could you please update that in each of the files?

**Review by cameronlee314 — COMMENTED** [review]
Thanks for finding these and updating this.

**Comment by bringhurst** [comment]
Remaking this over in https://github.com/apache/samza/pull/1539 since the merge got too ugly.

File: samza-api/src/main/java/org/apache/samza/checkpoint/CheckpointV1.java (hunk 1 above)
Comment (bringhurst) [reply samza-api/src/main/java/org/apache/samza/checkpoint/CheckpointV1.java:47]: Fixed in https://github.com/apache/samza/pull/1539

**Pull request closed without merging**